test/test_basic_logic.py

```diff
@@ -1699,21 +1699,20 @@ def sanity_check_data_frame(data_frame,
                             expected_flow_controlled_length,
                             expect_padded_flag,
                             expected_data_frame_pad_length):
-        """
-        ``data_frame`` is a frame of type ``hyperframe.frame.DataFrame``,
-        and the ``flags`` and ``flow_controlled_length`` of ``data_frame``
-        match expectations.
-        """
+    """
+    ``data_frame`` is a frame of type ``hyperframe.frame.DataFrame``,
+    and the ``flags`` and ``flow_controlled_length`` of ``data_frame``
+    match expectations.
+    """
 
-        assert isinstance(data_frame, hyperframe.frame.DataFrame)
+    assert isinstance(data_frame, hyperframe.frame.DataFrame)
 
-        assert(
-            (data_frame.flow_controlled_length ==
-             expected_flow_controlled_length)
-        )
-        if expect_padded_flag:
-            assert 'PADDED' in data_frame.flags
-        else:
-            assert 'PADDED' not in data_frame.flags
+    assert (
+        data_frame.flow_controlled_length == expected_flow_controlled_length
+    )
```

Comment:
Sorry, I should have been clearer. You can pull it onto a single line without the parentheses, i.e.,

```python
    assert data_frame.flow_controlled_length == expected_flow_controlled_length
```